os/mm/mm_heap/mm_partition_mgr.c

```diff
@@ -176,7 +217,19 @@ int8_t mm_allocate_ram_partition(uint32_t **start_addr, uint32_t *size)
 void mm_free_ram_partition(uint32_t address)
 {
 	mm_remove_app_heap_list((struct mm_heap_s *)address);
+#ifdef CONFIG_MM_PARTITION_HEAP
 	mm_free(&g_pheap, (void *)address);
+#else
+	if (g_mm_parts[0].start == address) {
+		g_mm_parts[0].size = 0;
+		g_mm_parts[0].start = g_ram_start;
```

Comment:
`g_mm_parts[0].start` is always `g_ram_start`.

Reply:
Removed.